File: modul-4/PRAK305-2210817210012-MuhammadRakaAzwar.c
#include<stdio.h>
int main()
{
    int detik1,detik2,menit,jam,hari,temp;
    int i = 1;
    while (i < 2){
            printf("");
            scanf("%d",&detik1);

            hari=detik1/86400;
            temp=detik1%86400;
            jam=temp/3600;
            temp=temp%3600;
            menit=temp/60;
            detik2=temp%60;
            if (detik1 >= 86400)
                printf("%d hari %02d:%02d:%02d\n",hari,jam,menit,detik2);
            else
                printf("%02d:%02d:%02d\n",jam,menit,detik2);
    }
}

File: modul-4/PRAK305-2210817210012-MuhammadRakaAzwar.py
i = 1
while i < 2:
    detik1 = int(input())
    hari=(detik1/86400)
    temp= detik1%86400
    jam=temp/3600
    temp=temp%3600
    menit=temp/60
    detik2=temp%60
    if (detik1 >= 86400):
        print("%d hari %02d:%02d:%02d" % (hari,jam,menit,detik2))
    else:
        print("%02d:%02d:%02d" % (jam,menit,detik2))